### DISCUSSION TITLE
While importing data through http data source...it is showing error (file size is too large to preview) through which I'm not able to do mapping

### DISCUSSION THREAD

**Discussion by gunnu1303** [pimcore/pimcore#14387] [Q&A / Support]
[
![Screenshot 2023-02-19 010114](https://user-images.githubusercontent.com/123451461/219884762-c2a316bc-e715-4694-a82f-1ca893547a61.png)
](url)

**Comment by SamyMP** [comment]
I think the preview file is supposed to contain one piece of what you are trying to import, not all your file. For example, instead of uploading a JSON having 10 000 objects, isolate one JSON payload into a file and upload this one for preview
